mnelab/dialogs/cropdialog.py

```diff
@@ -14,7 +14,9 @@ def __init__(self, parent, start, stop):
         self.setWindowTitle("Crop data")
 
         form = QFormLayout(self)
+        label_size = QSizePolicy.Expanding
```

Comment:
I think you forgot to import `QSizePolicy`.

Reply:
Yeah, I didn't check this line in.

Reply:
What do you think about this solution. Should we really prefer it over a QGridLayout?

Reply:
Not convinced yet. The vertical alignment seems to be OK, but the horizontal isn't right for items containing checkboxes. On macOS, the left column is by default right-aligned, this looks good if there are only labels, but with checkboxes not so much.

<img width="333" alt="Screen Shot 2020-02-17 at 17 03 34" src="https://user-images.githubusercontent.com/4377312/74669625-d8635c00-51a7-11ea-8559-7003c5499c25.png">

Reply:
Interesting, it perfectly fits on Ubuntu.
![image](https://user-images.githubusercontent.com/37848124/74670056-a999b580-51a8-11ea-90d8-2f041cc14e2b.png)

Reply:
Yes, it's because `QFormLayout` tries to adhere to each platform's interface guidelines, and on macOS the guidelines say that the labels must be right-aligned (https://doc.qt.io/qt-5/qformlayout.html). So I think we should use `QFormLayout` only for dialogs where the labels are just... labels.